core/settings.py

```diff
@@ -11,9 +11,8 @@
 SECRET_KEY = config('SECRET_KEY')
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = config('DEBUG', default=False, cast=bool)
+DEBUG = config('DEBUG', default=True, cast=bool)
 
-ALLOWED_HOSTS = config('ALLOWED_HOSTS', cast=Csv())
```

Comment:
@rodgomesc this is a security issue. Why take out this parameter?
ref: https://docs.djangoproject.com/en/2.1/ref/settings/#allowed-hosts

Reply:
sorry, I changed this for testing and forgot to update